tests/ui-fulldeps/session-diagnostic/diagnostic-derive.stderr

```diff
@@ -572,6 +556,24 @@ LL |     #[suggestion(no_crate_suggestion, code = "")]
    = help: to show a suggestion consisting of multiple parts, use a `Subdiagnostic` annotated with `#[multipart_suggestion(...)]`
    = help: to show a variable set of suggestions, use a `Vec` of `Subdiagnostic`s annotated with `#[suggestion(...)]`
 
+error[E0433]: failed to resolve: maybe a missing crate `core`?
```

Comment:
Why does the diagnostic change to this?

Reply:
I'm assuming this is generating a use of a variable name, but due to the previous error we did not actually generate the variable. I tried to fix it, but I can't make heads or tails of synstructure.

Reply:
I guess it's okay to regress this error for now.